dbms/src/Storages/KVStore/MultiRaft/Disagg/FastAddPeer.cpp

```diff
@@ -478,7 +493,8 @@ uint8_t ApplyFapSnapshotImpl(TMTContext & tmt, TiFlashRaftProxyHelper * proxy_he
         return false;
     }
     auto begin = checkpoint_ingest_info->beginTime();
-    if (kvstore->getRegion(region_id))
+    auto region = kvstore->getRegion(region_id);
+    if (region)
```

Comment:
In fact we don't. I explicitly defined that region because it will give us a clearer view that the region is not the same thing as region_to_ingest

Reply:
I think we can write it in this way
```
    if (auto region = kvstore->getRegion(region_id); region)
```

Reply:
I prefer your first advise that we just remove this `region`.
This line could be explicit for me, however, it could be confusing for others, may be add some comments is better.